blueprints/showbie-tooling/files/.eslintrc.js

```diff
@@ -2,15 +2,21 @@ module.exports = {
   root: true,
   parserOptions: {
     ecmaVersion: 8,
-    sourceType: 'module'
+    sourceType: 'module',
   },
-  extends: [
-    'eslint:recommended',
-    'plugin:ember-suave/recommended'
-  ],
   env: {
-    'browser': true
+    browser: true,
   },
+  extends: ['eslint:recommended', 'plugin:ember/recommended'],
   rules: {
-  }
-};
+    // Code style.
+    semi: ['error', 'always'],
+
+    // Custom Ember rules.
+    'ember/new-module-imports': 'warn',
```

Comment:
We should make these errors by default and then consuming applications can override if there's too many issues in the current codebase.

Reply:
Will do!